gst/isomp4/qtdemux.c

```diff
@@ -10750,13 +10750,15 @@ qtdemux_audio_caps (GstQTDemux * qtdemux, QtDemuxStream * stream,
     case 0x20736d:
     case GST_MAKE_FOURCC ('e', 'c', '-', '3'):
```

Comment:
I see it is restricted to 4 bytes. Scratch that.

Reply:
This code was already there and it's comparing 4 bytes read from a file to known fourccs (4-symbol identificator).
It is definitelly correct, otherwise it wouldn't enter this tree. I mean "ec-3" fourcc is written in the file.